server/views/autherror.njk

```diff
@@ -1,4 +1,13 @@
 {% extends "./partials/layout.njk" %}
+{% from "govuk/components/header/macro.njk" import govukHeader %}
+
+{% block header %}
```

Comment:
Authentication error page should use default GOVUK Header; not the One Login header (because you're not logged in to OL)